src/Microsoft.DotNet.Wpf/src/ReachFramework/Serialization/XpsImageSerializationService.cs

```diff
@@ -80,19 +76,9 @@ BitmapSource bitmapSource
             // a BitmapImage.  If it is not then we assume Png and use
             // that encoder.
             //
-            if (encoder == null)
-            {
-                if (Microsoft.Internal.AlphaFlattener.Utility.NeedPremultiplyAlpha(bitmapSource))
-                {
-                    encoder = new WmpBitmapEncoder();
-                }
-                else
-                {
-                    encoder = new PngBitmapEncoder();
-                }
-            }
-
-            return encoder;
+            return encoder ?? Microsoft.Internal.AlphaFlattener.Utility.NeedPremultiplyAlpha(bitmapSource)
```

Comment:
src\Microsoft.DotNet.Wpf\src\ReachFramework\Serialization\XpsImageSerializationService.cs(79,20): error CS0019: (NETCORE_ENGINEERING_TELEMETRY=Build) Operator '??' cannot be applied to operands of type 'BitmapEncoder' and 'bool'